### PR TITLE
project specific style file dc-#88

### PR DESCRIPTION
**Pull request by theisuru**
ebi-ait/data-catalogue#88

### CODE REVIEW

File: src/Form.module.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import './theme/theme.css';
+
+.submitButton {
+    color: var(--color-main);
+    background: var(--color-secondary);
+    font-family: var(--font-main);
+    border: 2px solid red;
```
Comment (sandsebi) [inline src/Form.module.css:7]: Maybe red can also be used as a variable?

File: src/theme/theme.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
+:root {
+    --color-main: #15a2e7;
+    --color-secondary: #deafe5;
+
+    --font-main: "DejaVu Sans", serif;
+}
+
+button {
+    --color-secondary: #cba64f;
```
Comment (amnonkhen) [inline src/theme/theme.css:9]: Are `--color-main` and the other variables representative css variable names or are they the ones controlling ag-grid as defined in the ag-grid docs?

There are two options:
1. project style file defines general variables names with what we allow to customise, like --color-main, and we use another internal style file to map these variables into framework specific styles, such as ag-grid or mui.
2. project style file contains the styling instructions for each framework we use

I favour No. 1.

**Review by amnonkhen — APPROVED** [review]

File: src/Form.module.css (hunk 1 above)
Comment (theisuru) [reply src/Form.module.css:7]: This was just there as an example. When we decide what we want here we can change these variables.

File: src/theme/theme.css (hunk 2 above)
Comment (theisuru) [reply src/theme/theme.css:9]: These are general variables. We can also add ag-grid varibles as you suggested. No. 1 option you sounds great.

**Pull request merged**